release-notes/next.yml

```diff
@@ -1,4 +1,10 @@
 releases:
+  42.20250427.1.0:
+    issues:
+      - text: "Docker Swarm DNS resolution failing in F42"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1933"
+      - text: "Occasional docker crashes on moby-engine-27.5.1-1.fc42"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1918"
```

Comment:
This one should also have https://github.com/coreos/fedora-coreos-tracker/issues/1939 because the fixed iptables is in `42.20250427.1.0`.

Reply:
done!